src/views/Definition/Definition.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { useStateValue } from '../../store/store.js';
+
+import { SearchBox } from '../../components';
+
+const Definition = () => {
+	const [{ dictionaryTitle, dictionaryIntroText }] = useStateValue();
+
+	return (
+		<>
+			<h1>{dictionaryTitle}</h1>
+			<p
+				dangerouslySetInnerHTML={{
+					__html: dictionaryIntroText,
```

Comment:
What is this doing here? There should not be any intro text except the home page. Can you make this like
```
<h1>Definition Title Here</h1>
<p>Definition Here</p>
```
Or really, just have nothing except your search box.

Reply:
Arghhh! Quite true, no intro text. Removed and put in placeholders for definition title and definition.

Reply:
Moved analyticsName to provider.